console/src/main/scripts/plugins/accessPortal/html/open_case.html

```diff
@@ -0,0 +1,3 @@
+<iframe id="openCaseWindow" width="100%" frameborder="0" src="/support.html#/case/new" scrolling="auto"
+        data-ng-controller="AccessPortal.AccessPortalMainController as accessPortalMain">
```

Comment:
Small nit. We have been removing the data-* to all the ng-* directives. This can be cleaned up later though.